.github/workflows/build-test-package-python.yml

```diff
@@ -34,11 +34,16 @@ on:
         description: 'JSON-formatted array of Python 3.x minor version wheel targets'
         required: false
         type: string
-        default: '["9","10","11"]'
+        default: '["9","10","11","12"]'
```

Comment:
This might not be needed, due to ABI compatibility. Matt should weigh in.

Reply:
This is wrong. We should not be trying build Python 3.12 packages. Our 3.11 packages support >= 3.11.